app/src/main/res/layout/exploration_fragment.xml

```diff
@@ -10,6 +10,7 @@
       android:id="@+id/state_fragment_placeholder"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
+      android:background="@color/exploration_activity_layout_background_color"
```

Comment:
Replace -> `activity` -> `fragment`


Also if all background colors are same then you can also create a shared one like `shared_player_background_color`

Reply:
Done